src/js/options.js

```diff
@@ -459,7 +533,7 @@ var ProvidersTab = {
     await LocalStore.setOne(StoreKey.SEARCH_PROVIDERS, providers);
 
     // Update UI according to this change.
-    mainConfigurationUpdated(true);
+    mainConfigurationUpdated();
```

Comment:
Here we should only update the contextual menu, however (since the `lazy` parameter was removed) this call is also re-initializing all forms, which is unnecessary.  
Same as other parts of the code that also passed `true` on this function.